charts/kargo/templates/common/cluster-roles.yaml

```diff
@@ -0,0 +1,15 @@
+{{- if .Values.rbac.installClusterRoles }}
```

Comment:
Thinking if this should be part of the api server template folder as this is needed by api server only?

Reply:
I'm good with that. Will move.

Reply:
Fixed in 3624f193f568efc041b6b8e657322cd816dc6289